### PR TITLE
chore: Add removed gha

### PR DESCRIPTION
**Pull request by gsanchezgavier**
Adds GithubActions removed in #103  and replaces the secrets to org level ones

### CODE REVIEW

File: cmd/nri-prometheus/versioninfo.json [hunk 1]
```diff
@@ -1,44 +0,0 @@
-{
-	"FixedFileInfo":
-	{
```
Comment (paologallinaharbur) [inline cmd/nri-prometheus/versioninfo.json:3]: Why removing this? Asking just in case

Comment (gsanchezgavier) [reply cmd/nri-prometheus/versioninfo.json:3]: Not sure exactly what this file is, but it was accidentally included in the [commit](https://github.com/newrelic/nri-prometheus/pull/103) , and i checked that the pipeline remove it in the [clean step](https://github.com/newrelic/nri-prometheus/blob/9ace92f5754afe4dfc5c0e82d2385d5ac8476475/build/release.mk#L18)

File: .github/workflows/helm.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  release:
+    types: [released]
+
+jobs:
+  create_helm_chart_pr:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create PR using Version Bump 
+      run: |
+        appVersion=${{github.event.release.tag_name}}
```
Comment (roobre) [inline .github/workflows/helm.yml:12]: I'm sorry I just could not hold myself
```suggestion
        appVersion=${{ github.event.release.tag_name }}
```

File: .github/workflows/helm.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        chartName=nri-prometheus
+        
+        curl -H "Accept: application/vnd.github.everest-preview+json" \
+        -H "Authorization: token ${{ secrets.VERSION_BUMP_TOKEN }}" \
```
Comment (roobre) [inline .github/workflows/helm.yml:16]: This is probably good as it is, but I wanted to double check: Would this work with `secrets.GITHUB_TOKEN`, which is automatically created by github actions? Maybe it won't if it's a different repo, but might be worth saving yet another secret to keep track of.

File: .github/workflows/prerelease.yml [hunk 4]
```diff
@@ -122,8 +122,8 @@ jobs:
       - name: Login to DockerHub
         uses: docker/login-action@v1 
         with:
-          username: ${{ secrets.DOCKER_USERNAME}}
-          password: ${{ secrets.DOCKER_PASSWORD}}      
+          username: ${{ secrets.FSI_DOCKERHUB_USERNAME}}
+          password: ${{ secrets.FSI_DOCKERHUB_TOKEN}}
```
Comment (roobre) [inline .github/workflows/prerelease.yml:126]:
```suggestion
          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
          password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
```

File: .github/workflows/push_pr.yml [hunk 5]
```diff
@@ -0,0 +1,96 @@
+name: Push/PR pipeline
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
```
Comment (roobre) [inline .github/workflows/push_pr.yml:8]: For other pipelines we use
```suggestion
on:
  push:
    tags:
      - v*
    branches:
      - main
  pull_request:
```
Might be a good opportunity to change it

File: .github/workflows/helm.yml (hunk 2 above)
Comment (gsanchezgavier) [reply .github/workflows/helm.yml]: sure i just check out this file from the last commit it was available. :)

File: .github/workflows/helm.yml (hunk 3 above)
Comment (gsanchezgavier) [reply .github/workflows/helm.yml:16]: I worked on this one and sadly it needs a Personal token kind :( . [More details](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows)

**Review by roobre — APPROVED** [review]
I haven't checked strictly whether this correspond exactly to what it was deleted but otherwise LGTM.

**Pull request merged**